### DISCUSSION TITLE
Possible to disable UI animation?

### DISCUSSION THREAD

**Discussion by HubKing** [tomasklaen/uosc#551] [Q&A]
Is it possible to have only two states, (1)UI completely-hidden when idling, and (2)UI fully-shown when the mouse moves over MPV, without in-between states like size/opacity animation, especially the timeline bar? 

The timeline bar currently gets gradually thicker as I move the mouse near the bottom, but personally I find it confusing.

**Comment by OmiChrome** [comment]
i actually want something like this ,  as 
- on some KEYBINDS  [show/hide] (UOSC)

coz only sometimes, when i dont want osc , uosc flashes up

**Comment by tomasklaen** [comment]
Use big `proximity_in` and `proximity_out` options.

**Reply by hooke007** [reply]
I think he was talking about the dup issue https://github.com/tomasklaen/uosc/issues/471
